zeppelin-frontend/README.md

```diff
@@ -0,0 +1,276 @@
+
+*.env*
+
+```
+SERVER_PROXY=http://localhost:8080
+```
+
+### Development server
+
+Run `npm start` for a dev server. Navigate to `http://localhost:4200/`. The app will automatically reload if you change any of the source files.
+
+### Build
+
+Run `npm build` to build the project. The build artifacts will be stored in the `dist/` directory.
+
+### Running unit tests
+
+Run `ng test` to execute the unit tests via [Karma](https://karma-runner.github.io).
+
+## Implementation Progress
+
+### Pages
+
+| Name | Route | Module     | UI |
+| ---  | ----- | ---------- | -- |
+| Home | `/`   | HomeModule |  Y |
+| Login | `/login`       | LoginModule |  Y |
+| Job Manager  | `/jobmanager`  | JobManagerModule |  Y |
+| Interpreter Setting   | `/interpreter` | InterpreterModule |  Y |
+| Notebook | `/notebook/{id}` | NotebookModule |  Y |
+| Notebook Repos | `/notebookRepos` |  |   |
+| Credential     | `/credential` |  |   |
+| Helium | `/helium` |  |  WIP  |
+| Configuration | `/configuration` |  |   |
+
+### Notebook Features
+
+| Feature | Note | Status |
+| ------  | ---- | ---- |
+| Files System  | Create/ Rename/ Import etc.  | Y |
+| Toolbar Actions  | The top toolbar actions   | Y |
+
+### Paragraph Features
+
+| Feature | Note | Status |
+| ------  | ---- | ---- |
+| Grid layout and resizable | | Y |
+| Code Editor | | Y |
+| Actions  | The Corresponding actions of the drop-down menu in the setting button | Y |
+| Actions(hot-keys)  | Support hot-keys for the actions  | WIP |
+| Publishable  | [publish paragraphs](http://zeppelin.apache.org/docs/0.8.0/usage/other_features/publishing_paragraphs.html)  |  |
+| Stream  |  |  |
+
+### Result Display
+
+| Type | Status |
+| ------  | ---- |
+| Dynamic Form  | Y |
+| Text  | Y |
+| Html  |  Y |
+| Table  |  Y |
+| Network  |  |
+
+### Table Visualization
+
+| Type | State |
+| ------ | ---- |
+| Line Chart  | Y |
+| Bard Chart  |  Y |
+| Pie Chart  |  Y |
+| Area Chart  |  Y |
+| Scatter Chart  | Y |
+
+### Helium Visualization
+
+| Type | Note | Status |
+| ------  | ---- | ---- |
+| Prototype | To verify the implementable prototype | Y |
+| Publish Dependencies | Just like [zeppelin-vis](https://github.com/apache/zeppelin/tree/master/zeppelin-web/src/app/visualization)  | WIP |
+| Example Projects |   | Y |
+| Development Documents |   | WIP |
+
+## Contributing
+
+### Dev Mode
+
+Follow the [Setup](#Setup) steps to starting the frontend service. The app will automatically reload if you change any of the source files.
+
+### Technologies
+
+Zeppelin-Frontend-Next is using Angular as the main Framework, before developing we hope highly recommended to have a good knowledge of [Angular](https://angular.io/) and [RxJs](https://github.com/ReactiveX/rxjs).
+
+In addition:
+
+- We use [G2](https://github.com/antvis/g2) visualization
+- We use [Lodash](https://lodash.com/) to process complex data
+- We use [Monaco Editor](https://github.com/microsoft/monaco-editor) to make code editor
```

Comment:
What are licences of these dependencies ? We need to update `LICENCE` file accordingly.

Reply:
Done